maxim/logger/gemini/async_client.py

```
@@ -148,41 +180,78 @@ async def send_message_stream(
 
     def __getattr__(self, name: str) -> Any:
+        """Get an attribute from the chat.
+
+        Args:
+            name: The name of the attribute.
+
+        Returns:
+            Any: The attribute.
+        """
         result = getattr(self._chats, name)
         return result
```

Ruff (0.11.9)

Unnecessary assignment to `result` before `return` statement

Remove unnecessary assignment

(RET504)